src/main/java/com/eerussianguy/blazemap/feature/maps/LayerButton.java

```diff
@@ -41,4 +41,8 @@ public void render(GuiGraphics graphics, int mx, int my, float partial) {
     public void checkVisible() {
         this.visible = host.getMapType() == parent;
     }
+
+    public void forceVisible() {
```

Comment:
What is this for?
Did I miss its usage, or did you not use it at all?

Reply:
This is for: https://github.com/LordFokas/BlazeMap/pull/46/files#diff-24d79d943443a3bb653bccf8c1b328970e1c6a1865a49c3e73612b32f917f2afR173

Basically, before this existed, I couldn't add the `LayerButton`s for the `MapType`s that weren't currently active because their visibility was only checked when first constructed. To have their visibility checked again would need to call `checkVisible()` on _every_ `LayerButton` when I wanted them to display.

Given I was already controlling the visibility of the buttons as a collection, I didn't want to waste CPU cycles having to check each button either every frame or every time the `MapType` changed. So instead, just gave the option to force the visibility to be `true` when creating and then don't need to check again!

Reply:
And of course pushing a new commit removes the line anchor 🤦‍♀️ 

Updated link: https://github.com/LordFokas/BlazeMap/pull/46/files#diff-9eb8d6a65ecef453ff1ebabe607a8a004ee4240a4cdad82074637b320b953076R168-R171

Reply:
It's acceptable until BME-82 tears half this stuff apart.